resolvers.py

```diff
@@ -19,11 +21,62 @@ def get_dates_and_price_check(
             **search_criteria
     ) -> List[DatesCheck]:
         validated_search_criteria = DatesAndPriceCheckInputValidator(**search_criteria)
-        results_list = []
+        houses_list = {}
```

Comment:
Since this is a dictionary data type and not a list, the variable name should be renamed as something like `houses_dict` instead to avoid confusion

Reply:
I would encourage `ids_to_houses_dict`, to be more explicit about what the key is.